website/docs/tutorials/07-testing/02-cheatcodes/reflect.md

```diff
@@ -0,0 +1,130 @@
+# `reflect`
+```python
+def reflect(self, ids: VmConsts) -> Reflector:
+
+class Reflector:
+    def get(self)
+```
+Loads specified Cairo object into Python type. ```get()``` can return:
+
+- int (for felt)
+- RelocatableValue (for pointer)
+- Cairo Struct (for more complex types)
+
+You can use it to print cairo data and compare complex structures.
+
+```cairo
+%lang starknet
+
+struct SimpleStruct:
+    member x: felt
+end
+
+@external
+func test_reflect_simple():
+    alloc_locals
+
+    local simple_struct: SimpleStruct = SimpleStruct(x=10)
+
+    %{
+        simple_struct = reflect(ids).simple_struct.get()
+        print(simple_struct)
+        # output:
+        # CairoStruct(
+        #     x=10
+        # )
+
+        assert simple_struct.x == 10 # true
+    %}
+
+    return()
+end
+```
+
+```cairo
+%lang starknet
+
+from starkware.cairo.common.registers import get_fp_and_pc
+
+@external
+func test_pointers():
+    alloc_locals
+
+    let (__fp__, _) = get_fp_and_pc()
+
+    local pointee: felt = 13
+    local ptr1: felt* = &pointee
+    local ptr2: felt* = &pointee
+    
+    %{
+        ptr1 = reflect(ids).ptr1.get()
+        ptr2 = reflect(ids).ptr2.get()
+
+        print(type(ptr1)) # RelocatableValue
```

Comment:
Can we provide output here to remain consistent? We can provide dummy pointer value